components/cloud/aws/src/s3.rs

```diff
@@ -0,0 +1,593 @@
+// Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
```

Comment:
`git mv` doesn't matter. The issue is that in addition to the move some changes to the file have been made. In the past I used to try to break these things up into multiple commits but I don't think anyone ever looks at individual commits no matter how helpful it is.

Reply:
You can diff this with the git CLI rather than github. Then you have the ability to tweak the diff parameters. Note that you can already do `?w=` on github, but you can't adjust `-M --find-renames`

```
git diff -w -M45% origin/master gregwebs/cloud-storage
```

Reply:
okay. cli is okay-ish but github has far better UI for side-by-side comparison and I can comment on it directly. 

Multiple commits would be especially helpful in this case though. They might not be useful other times.

Reply:
I briefly check the diff of s3.rs and gcs.rs and don't think I have further comment.